features/unattached_status.feature

```diff
@@ -128,45 +152,99 @@ Feature: Unattached status
+            ?<cis>( +<cis-available> +yes +no +Security compliance and audit tools)?
+            ?esm-apps      <esm-apps>  +yes  +yes  +UA Apps: Extended Security Maintenance \(ESM\)
+            esm-infra     <esm-infra> +yes  +yes  +UA Infra: Extended Security Maintenance \(ESM\)
+            fips          <fips>      +yes  +no   +NIST-certified core packages
+            fips-updates  <fips>      +yes  +no   +NIST-certified core packages with priority security updates
+            livepatch     <livepatch> +yes  +yes  +Canonical Livepatch service
+            ros           <ros>       +yes  +no   +Security Updates for the Robot Operating System
+            ros-updates   <ros>       +yes  +no   +All Updates for the Robot Operating System
+            ?<usg>( +<cis-available> +yes +no +Security compliance and audit tools)?
+            """
         When I do a preflight check for `contract_token` formatted as json
         Then stdout is a json matching the `ua_status` schema
         When I do a preflight check for `contract_token` formatted as yaml
         Then stdout is formatted as `yaml` and has keys:
-        """
-        _doc _schema_version account attached config config_path contract effective
-        environment_vars execution_details execution_status expires machine_id notices
-        services version simulated result errors warnings
-        """
+            """
+            _doc _schema_version account attached config config_path contract effective
+            environment_vars execution_details execution_status expires machine_id notices
+            services version simulated result errors warnings
+            """
+        When I verify that running `ua status --simulate-with-token invalid_token --format json` `as non-root` exits `1`
+        Then stdout is a json matching the `ua_status` schema
+        And I will see the following on stdout:
+            """
+            {"environment_vars": [], "errors": [{"message": "Invalid token. See https://ubuntu.com/advantage", "service": null, "type": "system"}], "result": "failure", "services": [], "warnings": []}
+            """
+        When I verify that running `ua status --simulate-with-token invalid_token --format yaml` `as non-root` exits `1`
+        Then stdout is formatted as `yaml` and has keys:
+            """
+            environment_vars errors result services warnings
+            """
+        And I will see the following on stdout:
+            """
+            environment_vars: []
+            errors:
+            - message: Invalid token. See https://ubuntu.com/advantage
+              service: null
+              type: system
+            result: failure
+            services: []
+            warnings: []
+            """
+        When I run `sed -i 's/contracts.can/contracts.staging.can/' /etc/ubuntu-advantage/uaclient.conf` with sudo
+        And I do a preflight check for `contract_token_staging_expired` formatted as json
```

Comment:
I checked that the command here is still returning `0`. I think it should return `1` instead. What do you think @orndorffgrant @CalvoM ?

Reply:
I agree that this should return non-zero